web/source/osk/visualKeyboard.ts

```diff
@@ -1261,6 +1351,20 @@ namespace com.keyman.osk {
       }
       // Add a faint border
       kbd.style.border='1px solid #ccc';
+
+      // Once the element is inserted into the DOM, refresh the layout so that proper text scaling may apply.
+      const refreshInterval = window.setInterval(function() {
+        let computedStyle = getComputedStyle(kbd);
+        if(computedStyle.fontSize) {
+          if(kbd.style.fontSize) {
+            // Preserve the new setting (provided by CSS)
+            kbdObj.fontSize = new ParsedLengthStyle(kbd.style.fontSize);
+          }
+          kbdObj.refreshLayout();
+          window.clearInterval(refreshInterval);
+        }
+      }, 10);
```

Comment:
Is there a chance this interval will never clear? At best, we need to document that the insertion of a keyboard into the DOM is async because of this.